assets/js/chooser.js

```diff
@@ -52,11 +52,16 @@ function selectChoice(kind, choice, extra) {
 // selectLanguage chooses a language.
 function selectLanguage(lang) {
     selectChoice("language", lang, function() {
+        var shellLanguages = ["bat", "batch", "batchfile", "powershell", "posh", "pwsh", "bash", "sh", "shell", "zsh"].map(
+            function(l) {
+                return "language-" + l;
+            }
+        );
         // In addition to the basic showing/hiding, highlight the right code blocks:
         $("code").each(function (i, e) {
             var classes = getElemClasses(e);
             for (var i = 0; i < classes.length; i++) {
-                if (classes[i].startsWith("language-") && classes[i] !== "language-bash" && classes[i] !== "language-sh") {
+                if (classes[i].startsWith("language-") && !shellLanguages.includes(classes[i])) {
```

Comment:
[IE11 doesn't support `includes`](https://caniuse.com/#feat=array-includes). While I don't think we need to concern ourselves much with IE these days (I'm not even sure how well the site renders in IE11 in general), switching to using `indexOf` instead would allow this to at least work in IE11 without any script errors.

I know we see errors on https://app.pulumi.com every once and a while due to someone browsing the site with IE and usage of `includes` there.

```suggestion
                if (classes[i].startsWith("language-") && !shellLanguages.indexOf(classes[i]) !== -1) {
```

Reply:
Wow, yeah, it's definitely been a while since I've written JS destined straight for the browser. I totally forgot that that kind of stuff still happens. Thanks for the suggestion!